packages/front-end/styles/theme.scss

```diff
@@ -0,0 +1,126 @@
+  --alert-info-border-color: #bee5eb;
+  --alert-info-color: #0c5460;
+
+  --alert-warn-background: #fff3cd;
+  --alert-warn-border-color: #ffeeba;
+  --alert-warn-color: #856404;
+
+  --alert-danger-background: #f8d7da;
+  --alert-danger-border-color: #f5c6cb;
+  --alert-danger-color: #721c24;
+
+  --border-color-100: #e0e2eb;
+  --border-color-200: #dee2e6;
+  --border-color-300: #cacaca;
+
+  --diff-removed-background: #ffeef0;
+  --diff-removed-background-highlight: #fdb8c0;
+  --diff-added-background: #e6ffed;
+  --diff-added-background-highlight: #acf2bd;
+}
+
+@mixin dark-theme {
+  color-scheme: dark;
+
+  --background-color: #{colors.$main-background-dark};
+  --surface-background-color: #{colors.$surface-background-dark};
+  --surface-background-color-alt: #{lighten(colors.$surface-background-dark, 4%)};
+  --table-row-hover-background-color: #{rgba(colors.$gray2, 0.15)};
+  --background-primary-light: #{darken(colors.$dark-purple, 13%)};
+
+  --text-color-main: #{colors.$white};
+  --text-color-table: #{lighten(colors.$gray2, 10%)};
+  --text-color-th: #{colors.$gray6};
+  --text-dark: #{colors.$gray6};
+  --text-link-color: #{lighten(#007bff, 20%)};
+  --text-link-hover-color: #{lighten(#007bff, 10%)};
+  --text-color-muted: #{colors.$gray-500};
+  --text-color-on-bright: #{colors.$black};
+  --text-color-primary: #{colors.$highlight-purple-light};
+  --text-color-on-background-primary-light: #{colors.$highlight-purple-light};
+
+  --form-input-group-background: #{colors.$gray-800};
+  --form-input-group-color: #{colors.$gray6};
+  --form-multivalue-background-color: #{colors.$highlight-purple-light};
+  --form-multivalue-text-color: #{colors.$black};
+  --form-switch-background: #{colors.$gray-800};
+  --form-switch-background-active: #{colors.$highlight-purple};
+
+  --alert-success-background: #{darken(#155724, 8%)};
+  --alert-success-border-color: #{darken(#155724, 0%)};
+  --alert-success-color: #d4edda;
+
+  --alert-info-background: #{darken(#0c5460, 8%)};
+  --alert-info-border-color: #{darken(#0c5460, 0%)};
+  --alert-info-color: #d1ecf1;
+
+  --alert-warn-background: #{darken(#856404, 15%)};
+  --alert-warn-border-color: #{darken(#856404, 8%)};
+  --alert-warn-color: #fff3cd;
+
+  --alert-danger-background: #{darken(#721c24, 8%)};
+  --alert-danger-border-color: #{darken(#721c24, 0%)};
+  --alert-danger-color: #f8d7da;
+
+  --border-color-100: #616161;
+  --border-color-200: #424242;
+  --border-color-300: #424242;
+
+  --diff-removed-background: #{rgba(#721c24, 0.35)};
+  --diff-removed-background-highlight: #721c24;
+  --diff-added-background: #{rgba(#155724, 0.35)};
+  --diff-added-background-highlight: #155724;
+}
+
+.theme--light {
+  @include light-theme();
+}
+
+.theme--dark {
+  @include dark-theme();
+}
+
+// OS default
+:root:not(.theme--light):not(.theme--dark) {
+  @media (prefers-color-scheme: light) {
+    @include light-theme();
+  }
+
+  @media (prefers-color-scheme: dark) {
+    @include dark-theme();
+  }
```

Comment:
`prefers-color-scheme` has been [available since 2019](https://caniuse.com/prefers-color-scheme) in Chrome, Edge, Firefox, Safari and Opera so I think it should be ok, but I've defaulted to light just in case.